src/types/config.ts

```diff
@@ -4,7 +4,8 @@ interface RelatedConfig {
   showOnPlaybackDone: boolean;
   showOnPlaybackPaused: boolean;
   playlistId: string | null;
-  entryList?: Array<string>;
+  entryList: Array<string>;
+  sourcesList: Array<any>;
```

Comment:
sourcesList should be Array<[ProviderMediaConfigSourcesObject](https://github.com/kaltura/playkit-js-providers/blob/master/flow-typed/types/media-config-sources.js)>

Reply:
not Array<KPMediaConfig> ?

Reply:
no. you should not get plugins in each item

Reply:
updated